web/docs/Frontend Starters/create-pantheon-decoupled-kit.md

```diff
@@ -108,6 +108,8 @@ npm init pantheon-decoupled-kit -- next-drupal next-drupal-umami-addon \
   # prevent the package manager from running
   # `install` after the project is created
   --noInstall
+  # CMS you are developing for. This flag is used to filter suggested generators by CMS compatibility. As of now, `wp` or `drupal` are valid arguments
+  --cmsType drupal
```

Comment:
I'm not sure the flag is necessary on this doc because the generators are already selected.